modules/covid_vaccine/spec/factories/expanded_registration_submissions.rb

```diff
@@ -90,6 +90,10 @@
       }
     end
 
+    # trait :enrollment_out_of_band do
```

Comment:
Is this meant to be commented out? If so, should it just be removed?

Reply:
thanks yep removed those comments.